openhtf/names.py

```diff
@@ -60,4 +59,3 @@
 TestPhase = openhtf.PhaseOptions
 PhaseResult = openhtf.PhaseResult
 Test = openhtf.Test
-UOM = openhtf.util.units.UOM
```

Comment:
I thought the plan was to move away from `names.py`, but maybe it's still useful as a todo list for when we move to the new way.